### PR TITLE
Added links to the Microsoft 365 Admin Center

### PR DESCRIPTION
**Pull request by DarqueWarrior**
As a new developer to M365 I had never visited nor knew how to find the Admin Center. I had to search the web for that information. I simply added a link to it in this document.
-------
cc: @o365devx

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[5cf6392](https://github.com/DarqueWarrior/office-dev-program-docs/commits/5cf63921e3af392e84130bb68f3018d392911511)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/install-sample-packs.md](https://github.com/DarqueWarrior/office-dev-program-docs/blob/patch-1/docs/install-sample-packs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C19%5Cea425683-0364-0229-639c-ec65004718e6%5CPullRequest%5C202006191255370607-64%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=yuS86VYWe7j3vqWZ%2F7Av0yPgkUmZupgZ9E5dO3AAUbs%3D&st=2020-06-19T12%3A51%3A54Z&se=2020-07-20T12%3A56%3A54Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ea425683-0364-0229-639c-ec65004718e6?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by DarqueWarrior** [comment]
#sign-off

File: docs/install-sample-packs.md [hunk 1]
```diff
@@ -88,11 +88,11 @@ No. These sample data packs are only compatible with the Microsoft 365 developer
 
 ## How can I see the sample data in my subscription?
 
-After you install the Users sample data pack, to see the users that were added, go to the **Microsoft 365 Admin Center** on your Microsoft 365 developer subscription. Under **Users**, select **Active users**. You will see the list of 16 users. You can select a user to view the associated metadata, including photos and licenses.
+After you install the Users sample data pack, to see the users that were added, go to the [**Microsoft 365 Admin Center**](https://admin.microsoft.com/) on your Microsoft 365 developer subscription. Under **Users**, select **Active users**. You will see the list of 16 users. You can select a user to view the associated metadata, including photos and licenses.
```
Comment (lindalu-MSFT) [inline docs/install-sample-packs.md:91]:
```suggestion
After you install the Users sample data pack, to see the users that were added, go to the [**Microsoft 365 Admin Center**](https://docs.microsoft.com/en-us/microsoft-365/admin/admin-overview/about-the-admin-center?view=o365-worldwide) on your Microsoft 365 developer subscription. Under **Users**, select **Active users**. You will see the list of 16 users. You can select a user to view the associated metadata, including photos and licenses.
```
Changed link to the docs help page. Not everyone has access to the admin center so this is the intermediary step to sign up.

File: docs/install-sample-packs.md [hunk 2]
```diff
@@ -88,11 +88,11 @@ No. These sample data packs are only compatible with the Microsoft 365 developer
...
 
 ![Screenshot of 16 users in the Microsoft 365 Admin Center, with metadata for a selected user](images/content-packs-07.PNG)
 
-After you install the Mail and Events sample pack, to see the sample data, in the **Microsoft 365 Admin Center**, choose **Show all** and then select **Exchange**. In the Exchange admin center, when you select **recipients**, you can see that each of the 16 users has mailboxes with mail and events added.
+After you install the Mail and Events sample pack, to see the sample data, in the [**Microsoft 365 Admin Center**](https://admin.microsoft.com/), choose **Show all** and then select **Exchange**. In the Exchange admin center, when you select **recipients**, you can see that each of the 16 users has mailboxes with mail and events added.
```
Comment (lindalu-MSFT) [inline docs/install-sample-packs.md:95]:
```suggestion
After you install the Mail and Events sample pack, to see the sample data, in the [**Microsoft 365 Admin Center**](https://docs.microsoft.com/en-us/microsoft-365/admin/admin-overview/about-the-admin-center?view=o365-worldwide), choose **Show all** and then select **Exchange**. In the Exchange admin center, when you select **recipients**, you can see that each of the 16 users has mailboxes with mail and events added.
```

**Review by lindalu-MSFT — CHANGES_REQUESTED** [review]

**Pull request merged**